backend/models/core/Capability.js

```diff
@@ -0,0 +1,63 @@
+import DB from "@backend/data_access/DB";
+import InsufficientPermissionsError from "@/components/errors/InsufficientPermissionsError";
+
+export default class Capability {
+  static async getCapabilitiesByLoggedInUser(
```

Comment:
I imagine this method to be a record to return an array of capabilities, without throwing any exception. The exception throwing should be done by the `assert...()` family of methods. As this is a model, its responsibility is to return data.